modules/textures/src/lib/utils/extract-mipmap-images.js

```diff
@@ -1,4 +1,4 @@
-export function sliceLevels(data, options) {
+export function extractMipmapImages(data, options) {
```

Comment:
Nit: Sadly `.d.ts` files only checks calls, not definitions. There is a way to import the types so we get the definitions checked against the types, see

https://github.com/visgl/loaders.gl/blob/master/modules/loader-utils/src/lib/binary-utils/buffer-utils.js#L1-L5